Jenkinsfile

```diff
@@ -1,28 +1,21 @@
 #!/usr/bin/env groovy
 
-boolean isProduction = env.BRANCH_NAME == "main"
+boolean isMain = env.BRANCH_NAME == "main"
 boolean isBeta = env.BRANCH_NAME == "beta"
-boolean isPreview = env.BRANCH_NAME == "develop"
 
-def bucket   = isProduction ? "production"
+def bucket   = isMain ? "preview"
```

Comment:
We do not have a production bucket anymore?

Reply:
see logic below